Wabbajack/View Models/Gallery/ModListGalleryVM.cs

```diff
@@ -56,8 +56,23 @@ public ModListGalleryVM(MainWindowVM mainWindowVM)
             : base(mainWindowVM)
         {
             MWVM = mainWindowVM;
-            GameType = ALL_GAME_TYPE;
 
+            // load persistent filter settings
+            settings = MWVM.Settings.Filters;
+            if (settings.IsPersistent)
+            {
+                GameType = !string.IsNullOrEmpty(settings.Game) ? settings.Game : ALL_GAME_TYPE;
+                ShowNSFW = settings.ShowNSFW;
+                OnlyInstalled = settings.OnlyInstalled;
+                Search = settings.Search;
+                // subscribe to save signal
+                MWVM.Settings.SaveSignal
```

Comment:
This subscription needs to not be inside the `IsPersistent` if statement.  Consider if the settings on-disk have persistence turned off on startup.  Now this constructor runs, and the subscription/callback to save to the settings won't be made at all, even if IsPersistent is turned on mid-run.